config/user.role.anonymous.yml

```diff
@@ -17,6 +16,5 @@ permissions:
   - 'access PDF version'
   - 'access comments'
   - 'access content'
-  - 'search content'
```

Comment:
Just checking this is intentional - and the changes in search permissions for admins and authenticated users too.

Reply:
Yes; these are core search permissions and not (as far as I know) used by search_api_db which is what GMS uses (though it uses the core search block as base for a custom block)